### PR TITLE
Pay 5597 Payment Status Update Ping 2

### PR DESCRIPTION
**Pull request by sayali-salunkhe**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by ranjeet-kumarhmcts — APPROVED** [review]

File: api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java [hunk 1]
```diff
@@ -257,6 +259,312 @@ public void return_404_Get_for_payment_failure() {
...
+
+        TelephonyCardPaymentsRequest telephonyPaymentRequest =
+                TelephonyCardPaymentsRequest.telephonyCardPaymentsRequestWith()
+                        .amount(new BigDecimal("550"))
+                        .ccdCaseNumber(ccdCaseNumber)
+                        .currency(CurrencyCode.GBP)
+                        .caseType("DIVORCE")
+                        .returnURL("https://www.moneyclaims.service.gov.uk")
+                        .build();
+
```
Comment (VamshiRudrabhatlaHMCTS) [inline api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java:338]: @sayali-salunkhe can you please change the payment method to cheque instead Telephony

File: api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java [hunk 2]
```diff
@@ -257,6 +259,312 @@ public void return_404_Get_for_payment_failure() {
...
+                .representmentDate("2022-10-10T10:10:10")
+                .build();
+        Response ping2Response = paymentTestService.paymentStatusSecond(
+                SERVICE_TOKEN_PAYMENT, "abcdefgh",
+                paymentStatusUpdateSecond);
+
+        assertEquals(ping2Response.getStatusCode(), NOT_FOUND.value());
+        assertEquals("No Payment Failure available for the given Failure reference", ping2Response.getBody().prettyPrint());
+    }
+
```
Comment (VamshiRudrabhatlaHMCTS) [inline api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java:567]: @sayali-salunkhe can you please add a negative scenario for 429 please?

File: api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java (hunk 1 above)
Comment (sayali-salunkhe) [reply api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java]: Updated

File: api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java (hunk 2 above)
Comment (sayali-salunkhe) [reply api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java:565]: Added new FT

Comment (VamshiRudrabhatlaHMCTS) [reply api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java:565]: Thanks

**Review by VamshiRudrabhatlaHMCTS — APPROVED** [review]

**Pull request merged**